openreview/venue/process/paper_recruitment_pre_process.js

```diff
@@ -0,0 +1,69 @@
+async function process(client, edit, invitation) {
+
+  const committeeInvitedId = invitation.content.committee_invited_id.value
+  const committeeName = invitation.content.committee_name.value
+  const inviteAssignmentInvitation = invitation.content.invite_assignment_invitation_id.value
+  const hash_seed = invitation.content.hash_seed.value
```

Comment:
Not sure if these properties will always be in `content`, but if they aren't then I suggest using optional chaining too.

```suggestion
  const committeeInvitedId = invitation.content.committee_invited_id?.value
  const committeeName = invitation.content.committee_name?.value
  const inviteAssignmentInvitation = invitation.content.invite_assignment_invitation_id?.value
  const hash_seed = invitation.content.hash_seed?.value
```